examples/resourceServer/server/methods.js

```diff
@@ -0,0 +1,8 @@
+
+
+Meteor.methods({
+  // AUTH FLOW - Step C1.1
+  'oauth/addclient':function (newclient){
+    console.log('oauth/addclient', newclient);
```

Comment:
Upon ratification of previous discussion, this will have to be built out. It will be pretty simple.